avm/res/web/site/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -17,6 +17,12 @@ param resourceLocation string = deployment().location
 @description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
 param serviceShort string = 'wswaf'
 
+@description('Optional. Tags of the resource.')
+param tags object = {
```

Comment:
While it's a good thought, please move the value down into the test as a value. Reason being that the `Set-AVMModule` script carves the entire test block out of this file, formatts it and puts it into the readme. Anything that is not inside the test block is hence ignored. However, as we want to show as much as possible in the `Usage Examples` it's pretty much the opposite of what we'd want. 💪

Reply:
Update made and README updated per c1ff5453e7602fcd17b5dfd13814a9acd3f17c8f